Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -119,6 +123,13 @@ def build_iterator(self) -> List:
                                 f'Check your Server URL parameter.\n\n{err}')
             except requests.exceptions.HTTPError as err:
                 demisto.debug(f'Got an error from {feed_url} while fetching indicators {(str(err))} ')
+                if err.response.status_code == 503:
+                    raise Exception(f'The service located at {feed_url} is unavailable while fetching '
+                                    f'indicators {(str(err))} ')
+                elif err.response.status_code == 400 and region == GERMANY:
```

Comment:
after this if you should have a general raise for al the other cases, so we wold not silence the other http errors. we should fail on the also.